pegasus/migrations/109_add_hoc_activity_source.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+  up do
+    add_column :hoc_activity, :source, String, size: 50
+    add_index :hoc_activity, :source
+  end
+
+  down do
+    drop_index :hoc_activity, :source
+    drop_column :hoc_activity, :source
+  end
+end
```

Comment:
Do we have any idea how long this migration will take? My concern is that we are going to have to jump through hoops for this to complete successfully.